### PR TITLE
[COD-21] Create group frontend

### PR DESCRIPTION
**Pull request by SilaEwGenii**
https://legolas.atlassian.net/browse/COD-21
Add navbar and page created group

### CODE REVIEW

File: frontend/pnpm-lock.yaml [hunk 1]
```diff
@@ -0,0 +1,2281 @@
+lockfileVersion: 5.4
```
Comment (npupko) [inline frontend/pnpm-lock.yaml:1]: Please, remove `pnpm-lock.yaml`. We're using docker with `yarn` now

**Review by npupko — COMMENTED** [review]
Please, add `.DS_Store` to gitignore and remove this file from git

File: frontend/src/routes/groups/new/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+					class="w-full rounded-lg border-gray-200 p-4 pr-12 text-sm shadow-sm invalid:border-red-500"
+					placeholder="Input emails of users you want to invite"
+				/>
+			</div>
+		</div>
+
+		<div class="flex items-center justify-between">
+			<button
+				type="submit"
+				class="ml-3 inline-block rounded-lg bg-rose-500 px-5 py-3 text-sm font-medium text-white"
```
Comment (Set27) [inline frontend/src/routes/groups/new/+page.svelte:41]: delete margin

**Review by npupko — APPROVED** [review]
LGTM

**Pull request merged**